src/components/income/W2JobInfo.tsx

```diff
@@ -47,18 +67,31 @@ const toIncomeW2 = (formData: IncomeW2UserInput): IncomeW2 => ({
   medicareIncome: parseInt(formData.medicareIncome),
   fedWithholding: parseInt(formData.fedWithholding),
   ssWithholding: parseInt(formData.ssWithholding),
-  medicareWithholding: parseInt(formData.medicareWithholding)
+  medicareWithholding: parseInt(formData.medicareWithholding),
+  state: formData.state,
+  stateWages: parseInt(formData.stateWages),
+  stateWithholding: parseInt(formData.stateWithholding)
 })
 
 const toIncomeW2UserInput = (data: IncomeW2): IncomeW2UserInput => ({
+  ...blankW2UserInput,
   ...data,
   income: data.income.toString(),
   medicareIncome: data.medicareIncome.toString(),
   fedWithholding: data.fedWithholding.toString(),
   ssWithholding: data.ssWithholding.toString(),
-  medicareWithholding: data.medicareWithholding.toString()
+  medicareWithholding: data.medicareWithholding.toString(),
+  state: data.state,
+  stateWages: data.stateWages?.toString() ?? '',
+  stateWithholding: data.stateWithholding?.toString() ?? ''
 })
 
+const boxLabel = (box: string, description: string): ReactElement => (
```

Comment:
since other pages use a box with the 'strong' label, I think this should be moved to be in the input directory